app/src/androidTest/java/com/hedvig/app/feature/home/ActiveInFutureAndTerminatedInFutureTest.kt

```diff
@@ -30,7 +30,7 @@ class ActiveInFutureAndTerminatedInFutureTest {
     val mockServerRule = ApolloMockServerRule(
         LoggedInQuery.QUERY_DOCUMENT to apolloResponse {
             success(
-                LOGGED_IN_DATA_WITH_REFERRALS_FEATURE_ENABLED
+                LOGGED_IN_DATA_WITH_KEY_GEAR_FEATURE_ENABLED
```

Comment:
This does not look correct?

Reply:
Before we had `val LOGGED_IN_DATA_WITH_REFERRALS_FEATURE_ENABLED = LoggedInDataBuilder().build()
` which resulted in only the key gear feature being enabled due to the the constructor having `data class LoggedInDataBuilder(
    val features: List<Feature> = listOf(
        Feature.KEYGEAR
    ),...)`. Thats why i renamed it to this. So maybe it was named wrong before, or im missing something

Reply:
Interesting! I think that the name of the test data item was probably incorrect then - it's been used pretty much exclusively to represent a user which has referrals enabled.

Did you run the tests before pushing this commit? I think it probably breaks a lot of them.

Reply:
Will take a look to see if this brakes something